google/cloud/security/inventory/transform_util.py

```diff
@@ -39,6 +39,16 @@ def flatten_projects(projects):
     for project in (project for d in projects \
                     for project in d.get('projects', [])):
         project_json = json.dumps(project)
+
+        project_id = project.get('projectId')
+        lifecycle_state = project.get('lifecycleState')
+        parent = project.get('parent', {})
+
+        if lifecycle_state != 'ACTIVE':
```

Comment:
Ah, I should've read the description.  :)  I feel that this filtering should be done upstream in the get_projects(), so that if anything else calls get_projects(), they would not have to do the filtering again.